src/components/Home/styles.module.scss

```
@@ -4,9 +4,82 @@
 }
 .hr {
     display: block;
-    margin-top: 5px;
-    margin-bottom: 5px;
+    margin-top: 8px;
+    margin-bottom: 8px;
     background-color: gray;
     opacity: 0.3;
     height: 2px;
-}
\ No newline at end of file
+}
+
+.cont1 {
+    width:100%;
+    height: 70%;
+    display: flex;
+    align-items: flex-end;
+}
+
+.containerHome {
+    width: 40%;
+    font-size: 30px;
+}
+
+.title {
+    color: blue;
+    font-size: 10vh;
+}
+
+.cont2 {
+    width: 100%;
+    height: 30%;
+}
+
+.containerButton {
+    height: auto;
+    width: auto;
+    margin-top: calc(2% + 28px);
+}
+
+
+.button1 {
+    background-color:#FF0080;
+    color: white;
+    font-size: 20px;
+    border-radius: 5px;
+    margin-right: 25px;
+    border-color: #FF0080;
+    width: 250px;
+    height: 64px;
+    transition: all 1.2s;
+    &:hover {
+        transform: translateY(-5px);
+        box-shadow: 0 30px 30px #ee9ca1;
+        cursor:pointer;
+    }
+}
+
+.button2 {
+    background-color: blue;
+    color: white;
+    font-size: 20px;
+    border-radius: 5px;
+    border-color: blue;
+    width: 250px;
+    height: 64px;
+    transition: all 1.2s;
+    &:hover {
+        transform: translateY(-5px);
```

Ten en cuenta el comentario de arriba respecto a los espaciados y a la duración de las transiciones.

```suggestion
    transition: all 1.2s;

    &:hover {
        transform: translateY(-5px);
```